### DISCUSSION TITLE
bug report

### DISCUSSION THREAD

**Discussion by ryuk8000** [WalletConnect/walletconnect-monorepo#2472] [Developer Support]
https://lab.web3modal.com/v1ManagedReact

How to repoduce bug. 
Go to this link.

On desktop First click on zerion wallet and now close the zerion prompt window before connection takes place. 
Now try connecting metamask wallet we don't get any connection prompt

**Comment by finessevanes** [comment]
Closing this since that package will be deprecated in less than two weeks.
